sympy/assumptions/facts.py

```diff
@@ -11,29 +11,55 @@
 from sympy.core.cache import cacheit
 from sympy.core.symbol import Symbol
 from sympy.logic.boolalg import (to_cnf, And, Not, Implies, Equivalent,
-    Exclusive,)
+    Exclusive, Or,)
 from sympy.logic.inference import satisfiable
 
 
-@cacheit
-def get_composite_predicates():
```

Comment:
No harm in leaving this here, is there? This can be called from `decompose_predicate` rather than embedding the non-cached version there. That way anyone that wants this mapping can get it -- kind of like being able to retrieve a list of all the old-style assumptions.

Reply:
Any thoughts on this?